packages/zero-cache/src/server/otel-start.ts

```diff
@@ -3,82 +3,92 @@ import {OTLPMetricExporter} from '@opentelemetry/exporter-metrics-otlp-http';
 import {OTLPLogExporter} from '@opentelemetry/exporter-logs-otlp-http';
 import {NodeSDK} from '@opentelemetry/sdk-node';
 import {
-  ATTR_SERVICE_NAME,
-  ATTR_SERVICE_VERSION,
-} from '@opentelemetry/semantic-conventions';
+  detectResources,
+  envDetector,
+  processDetector,
+  hostDetector,
+  resourceFromAttributes,
+  defaultResource,
+} from '@opentelemetry/resources';
+import {ATTR_SERVICE_VERSION} from '@opentelemetry/semantic-conventions';
 import {
   PeriodicExportingMetricReader,
   ConsoleMetricExporter,
 } from '@opentelemetry/sdk-metrics';
-import {resourceFromAttributes} from '@opentelemetry/resources';
-import {NoopSpanExporter} from '../../../otel/src/noop-span-exporter.ts';
-import {NoopMetricExporter} from '../../../otel/src/noop-metric-exporter.ts';
 import {version} from '../../../otel/src/version.ts';
 import {
   BatchLogRecordProcessor,
   LoggerProvider,
   type LogRecordProcessor,
 } from '@opentelemetry/sdk-logs';
 import {logs} from '@opentelemetry/api-logs';
+import {getNodeAutoInstrumentations} from '@opentelemetry/auto-instrumentations-node';
 
-export type OtelEndpoints = {
-  traceCollector?: string | undefined;
-  metricCollector?: string | undefined;
-  logCollector?: string | undefined;
-};
+class OtelManager {
+  private static _instance: OtelManager;
+  private _started = false;
```

Comment:
nit: preference is for the JS `#` for private fields
```suggestion
  static #instance: OtelManager;
  #started = false;
```